src/librustdoc/html/render/mod.rs

```diff
@@ -4108,7 +4101,7 @@ fn item_typedef(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, t: &clean::T
 }
 
 fn item_foreign_type(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item) {
-    writeln!(w, "<pre class=\"rust foreigntype\">extern {{");
+    w.write_str("<pre class=\"rust foreigntype\">extern {");
```

Comment:
You forgot a backline here too.

Reply:
Might be systematic 😅 I'll go through these myself as well. Thanks!